### PR TITLE
wayfire: 0.6.0 -> 0.7.0

### PR DESCRIPTION
**Pull request by alyssais**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/window-managers/wayfire/wcm.nix [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-  version = "0.6.0";
+  version = "0.7.0";
 
-  src = fetchFromGitHub {
-    owner = "WayfireWM";
-    repo = "wcm";
-    rev = "v${version}";
-    sha256 = "1b22gymqfn0c49nf39676q5bj25rxab874iayiq31cmn14r30dyg";
+  src = fetchurl {
+    url = "https://github.com/WayfireWM/wcm/releases/download/v${version}/wcm-${version}.tar.xz";
```
Comment (SuperSandro2000) [inline pkgs/applications/window-managers/wayfire/wcm.nix:10]: Whats the difference between this download and the github repo?

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112326` at 39c2d86d run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>wayfire</li>
    <li>wcm</li>
    <li><a href="https://gist.github.com/babe0c91250c4f92d2a7da3952ee4ddc">wf-config</a></li></li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112326` at 39c2d86d run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>wayfire</li>
    <li>wcm</li>
    <li><a href="https://gist.github.com/b801e0511c12d986d3d833c83c4d073d">wf-config</a></li></li>
  </ul>
</details>

File: pkgs/applications/window-managers/wayfire/wcm.nix (hunk 1 above)
Comment (alyssais) [reply pkgs/applications/window-managers/wayfire/wcm.nix:10]: I don’t know, but there’d be no need to manually generate and upload a tarball if they didn’t want people to use it.

**Comment by rmcgibbo** [comment]
Hi @alyssais -- Sorry for the bot posts here. I know you asked Sandro not to post these reports on your PRs  if there was no build failure, and I was trying to honor that request as well, but there was a bug in my logic. It's now fixed, so these "everything is peachy" comments won't be posted on yours PRs again.

**Comment by alyssais** [comment]
> Hi @alyssais -- Sorry for the bot posts here. I know you asked Sandro not to post these reports on your PRs if there was no build failure, and I was trying to honor that request as well, but there was a bug in my logic. It's now fixed, so these "everything is peachy" comments won't be posted on yours PRs again.

No worries! Thanks for being so considerate! :)

**Pull request merged**